config/vufind/httpd-vufind.conf

```diff
@@ -79,8 +79,8 @@ Alias /vufind /usr/local/vufind/public
 
   # This line specifies additional Zend Framework 2 modules to load after the standard VuFind module.
   # Multiple modules may be specified separated by commas.  This mechanism can be used to override
-  # core VuFind functionality without modifying core code.
-  #SetEnv VUFIND_LOCAL_MODULES VuFindLocalTemplate
+  # core VuFind functionality without modifying core code or to enable the VuFindApi module.
+  #SetEnv VUFIND_LOCAL_MODULES VuFindApi,VuFindLocalTemplate
```

Comment:
This change will break the installer's ability to enable custom modules (see https://github.com/vufind-org/vufind/blob/master/install.php#L515). I'd be inclined to just enable the API module as part of the default list (https://github.com/vufind-org/vufind/blob/master/config/application.config.php#L5) and worry about coming up with a mechanism to disable core modules through configuration if the need arises. (This would be consistent with how VuFindAdmin is currently handled, and following that precedent probably makes life easier for most people).